### DISCUSSION TITLE
How to join farming pools?

### DISCUSSION THREAD

**Discussion by Lovejane-N** [Chia-Network/chia-blockchain#551] [Q&A] [answered]
Hi, is there a chance to join farming pool now ? Or we need to wait for the pool protocol. The total space increases so fast and the individual users is getting harder and harder to actually farm the blocks.

**Comment by hoffmang9** [comment, marked as answer]
Bram recently posted on this - [Pools in Chia](https://www.chia.net/2020/11/10/pools-in-chia.html).

-Gene

**Reply by room101-dev** [reply]
Already here its called HPOOL (dot) COM works great and has the highest payout lowest fee's and is the longest running chia plotter on earth.

There website is in english, they payout every 5-15 minutes 24/7, very professional large chinese pooling  company. Know the business, and have 100's of 1,000's of customers. They collect the majority of chia mined on earth. They have been in business for years, and they never ripoff their customers.l

**Comment by hoffmang9** [comment]
Also - we are about to switch to [new consensus](https://docs.google.com/document/d/1tmRIb7lgi4QfKkNaxuKOBHRmwbVlGL4f7EsBDr_5xZE/edit) and the math is likely more favorable than you are worried about - https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#i-have-only-10-tb-will-i-ever-win-xch-on-mainnet

**Comment by chiaseednetwork** [comment]
The downside is that plots being made today won’t be able to switch pools in the future without replotting.

**Reply by AnthonyDee123** [reply]
Really? So plotting now and waiting for pools to farm is can't happen? Do you know this for sure?

**Reply by sienar1** [reply]
The portable plots will be different. Pooling current plots apparently requires sharing your private key

**Comment by PPGOli** [comment]
What do you think about the Chinese Chia "pool" solution like hpool.com?
Executable posted here:
https://github.com/hpool-dev/chia-miner/releases
Without source code of course...

But according to chiaexplorer.com their address
xch1f0ryxk6qn096hefcwrdwpuph2hm24w69jnzezhkfswk0z2jar7aq5zzpfj
is the biggest earner, apparently representing 31% of the network space...

Thank you for your thoughts

-Oliver

**Comment by brezapour81** [comment]
Hey All, 

We at https://seedpool.org will be launching our UK hosting Chai Pool soon. It will be using the official Chia Protocol once releases.

Check it out if you are interest.

**Reply by DuDlikSPB** [reply]
Hi, any info when it will be realised?

**Reply by brezapour81** [reply]
17th May 2021.

**Reply by DuDlikSPB** [reply]
What 17th may? You will start the pool or just pool code will be realised?

**Reply by sfahadshahzad** [reply]
any update ?

**Reply by brezapour81** [reply]
The Chia Official Protocol date has been delayed which means will be expecting the details end of May from Chia.net.

Seedpool.org will likely be up shortly after.

Keep a eye on the site and subscribe to be notified as soon as we are up and running.

**Reply by KhaledElzarw** [reply]
Thank you very much? 
do you know when is the expected launch date and can you please share the official announcement link?

**Reply by IvanMurzak** [reply]
For farming in the pool, should we re-plotting?

**Reply by joelrb** [reply]
> For farming in the pool, should we re-plotting?

I think so. It says in the guide we should re-plot with portable ones which means these plots can be used for both pool farming and solo farming.

Will I need to replot to use the official pooling protocol?
Yes. Anyone who wants to join a pool will need to create new K32 portable plots. This new plot format allows you to switch between pools and self-pooling with a cooldown of 30 minutes (rules set by pool) between each switch. Each switch between pools will require a transaction with a smart contract on the blockchain. Our recommendation is to slowly replace your existing plots with portable plots one by one, so you still have a chance to win XCH while you convert to all portable plots.

**Reply by IvanMurzak** [reply]
@joelrb thanks, how to make a portable plot? Should I use some specific soft for that or just enough to use the main Chia blockchain app?

**Reply by joelrb** [reply]
> @joelrb thanks, how to make a portable plot? Should I use some specific soft for that or just enough to use the main Chia blockchain app?

It's not yet available. Support for portable plots is scheduled on or before May 31. Stay tuned.

**Reply by MavrikiDev** [reply]
> It's not yet available. Support for portable plots is scheduled on or before May 31. Stay tuned.

Thanks for the information.  I'm analyzing which pool join to.

**Comment by unitedchia** [comment]
Hi,
we are the pool [unitedchia.com](https://unitedchia.com). Feel free to stop by and sign up.

We will provide a transparent and reliable pool using the official pooling protocol. 

The launch of the pool will be right after the release of the protocol. We have already developed an infrastructure, that simply waits to be launched.

See you around.

**Reply by padik11** [reply]
What size of pool do you estimate at the beginning?

**Reply by YoCoinYOCO** [reply]
Do you have a promo code available? I would like to sign up, I have close to 40 plots & growing.

**Reply by krabradosty** [reply]
@unitedchia Chia wiki says that protocol will be released to Testnet by end of May. Do you have any estimation for Mainnet? When should we expect it? Any insights?
